cfgmgr/teammgrd.cpp

```diff
@@ -17,13 +18,24 @@ bool gLogRotate = false;
 ofstream gRecordOfs;
 string gRecordFile;
 
+bool received_sigterm = false;
+
+void sig_handler(int signo)
```

Comment:
>signo [](start = 21, length = 5)

check signo == SIGTERM first?

Reply:
The Signal handler is only registered only for SIGTERM now. The handler will be called only on SIGTERM signal.